### ISSUE TITLE
IGV vcf file visualization: Error accessing resource

### ISSUE THREAD

**Issue by Dnyane** [igvteam/igv.js#406]
Hello,
I am new to using IGV tools. I want to use the IGV tool for vcf file visualization on remote linux server. 
I tried running 1kg-vcf.html  for this purpose. 
I have a fasta, bed and indexed bed file version of a bacterial genome => GCA_000196095.1_ASM19609v1_cds_from_genomic (Vibrio parahaemolyticus RIMD starin)
I have an example vcf file 8Filtered.vcf which I want to visualize using IGV.
Desktop version seems to work fine for my data but I want to use IGV to develop a web resource (Database) which can provide variation viewing.

This is my script:

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="//igv.org/web/img/favicon.ico">

    <title>IGV - Integrative Genomics Viewer</title>

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.5.7/es5-shim.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/es6-shim/0.35.3/es6-shim.js"></script>

    
    <link rel="stylesheet" type="text/css"
          href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/themes/smoothness/jquery-ui.css"/>

    
    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">

    
    <link rel="stylesheet" type="text/css" href="https://igv.org/web/beta/igv.css">

    
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>

    
    <script type="text/javascript" src="https://igv.org/web/test/multilocus/igv-multilocus.js"></script>

    <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
                    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

        ga('create', 'UA-92237317-1', 'auto');
        ga('send', 'pageview');

    </script>

</head>

<body>

<p>

<h1>1000 Genomes VCF file</h1></p>

<div id="igvDiv" style="padding-top: 10px;padding-bottom: 10px; border:1px solid lightgray"></div>

<script type="text/javascript">

    var div = document.getElementById('igvDiv');
    var options = {
        reference: {
            id: "hg19",
            fastaURL: "GCA_000196095.1_ASM19609v1_cds_from_genomic.fna",
        },
        locus: "myc",
        tracks: [
            {
                name: "Genes",
                type: "annotation",
                format: "bed",
                sourceType: "file",
                url: "GCA_000196095.1_ASM19609v1_genomic.bed.gz",
                indexURL: "GCA_000196095.1_ASM19609v1_genomic.bed.gz.tbi",
                order: Number.MAX_VALUE,
                visibilityWindow: 300000000,
                displayMode: "EXPANDED"
            },

            {
                name: "Phase 3 WGS variants",
                format: "vcf",
                url: "8Filtered.vcf.gz",
                indexURL:  "8Filtered.vcf.gz.tbi",
                type: "variant"
            }
        ]

    };
    var browser = igv.createBrowser(div, options);

</script>

</body>

</html>

This is my output:

Error Accessing Resource : GCA_000196095.1_ASM19609v1_cds_from_genomic.fna.fai Status:0

I have All the files including  this script  as well as fasta fna, fai files of genome in the same directory.

**Comment by turner** [comment]
Hi,

Try replace this line:
```
<!-- IGV JS-->
<script type="text/javascript" src="https://igv.org/web/test/multilocus/igv-multilocus.js"></script>

```
with this line:

```
<!-- IGV JS-->
<script type="text/javascript" src="https://igv.org/web/beta/igv-beta.min.js"></script>

```

**Comment by jrobinso** [comment]
Correction,  replace 

<script type="text/javascript" src="https://igv.org/web/test/multilocus/igv-multilocus.js"></script>

with

<script type="text/javascript" src="https://igv.org/web/release/1.0.9/igv-1.0.9.js"></script>

**Comment by jrobinso** [comment]
Hi,

First, replace 
<script type="text/javascript" src="https://igv.org/web/test/multilocus/igv-multilocus.js">
 with 
<script type="text/javascript" src="https://igv.org/web/beta/igv-beta.min.js">

Second, what type of webserver are you serving this from?  Server requirements are documented here: https://github.com/igvteam/igv.js/wiki/Data-Server-Requirements.   Most web servers meet these requirements, but some might need configuration to enable range-byte requests.   For an out-of-the box solution see our flask server project:  https://github.com/igvteam/igv.js-flask

**Comment by jrobinso** [comment]
Closing due to lack of response
